src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -126,12 +141,27 @@ private void setAccelerator(MenuItem menuItem, KeyCombination keyCombination) {
      * Fills up all the placeholders of this window.
      */
     void fillInnerParts() {
+
+        friendText.setText("Friends");
```

Comment:
Something to be done next milestone is probably to implement this with a .css stylesheet instead